Gopkg.lock

```diff
@@ -2,14 +2,175 @@
 
 
 [[projects]]
+  branch = "master"
+  name = "github.com/Azure/go-ansiterm"
+  packages = [".","winterm"]
+  revision = "fa152c58bc15761d0200cb75fe958b89a9d4888e"
+
+[[projects]]
+  name = "github.com/Microsoft/go-winio"
```

Comment:
@enoodle @simon3z thoughts on this? personally my preference is to include more dependencies to allow for possible edge cases and preclude technical debt (i.e. having to come in and change this back to `alltransports` at a future time).